Makefile

```diff
@@ -37,7 +37,11 @@ CMAKE_ARGS ?=
 VERBOSE ?=
 FORCE_CLANG_FORMAT ?=
 
-pyextsuffix := $(shell python3-config --extension-suffix)
+WHICH_PYTHON := $(shell which python)
```

Comment:
It mush be `python3`.  We should not allow any possibility of running python 2.

Reply:
This is fair. I will update it.